src/event.js

```diff
@@ -0,0 +1,20 @@
+import Logic from './logic';
+
+export default class Event {
```

Comment:
This is a problematic class name because you are overwriting the built-in Event class. Try to rename it to for example `UserEvent`.